src/Ng/CommandHelpers.cs

```diff
@@ -79,7 +83,7 @@ private static ISecretInjector GetSecretInjector(IDictionary<string, string> arg
 
         public static RegistrationStorageFactories CreateRegistrationStorageFactories(IDictionary<string, string> arguments, bool verbose)
         {
-            StorageFactory legacyStorageFactory;
+            NuGet.Services.Metadata.Catalog.Persistence.StorageFactory legacyStorageFactory;
```

Comment:
>NuGet.Services.Metadata.Catalog.Persistence.StorageFactory [](start = 12, length = 58)

Please don't put the type's full-name here. If there's a naming conflict, add an alias like `CatalogStorageFactory`.